### DISCUSSION TITLE
libpmem-dev not supported by brew?

### DISCUSSION THREAD

**Discussion by williamwu971** [Homebrew/discussions#1680] [Everyday usage]
As titled, after browsing the available packages, I didn't find any packages that can replace the libpmem-dev package for apt install. Is there an alternative? I need the libpmem.h file for my project.
Thanks in advance ;)

**Comment by SMillerDev** [comment]
You can follow the guide here: https://docs.brew.sh/Adding-Software-to-Homebrew
